src/app/shared/components/icon-button/icon-button.component.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
We should get in the happit of deleting empty SCSS files. We can always add them again if needed.

Reply:
Good point, i deleted the file!